saul-core/src/main/java/edu/illinois/cs/cogcomp/saul/test/TestContinuous.java

```diff
@@ -4,7 +4,7 @@
   * Developed by: The Cognitive Computations Group, University of Illinois at Urbana-Champaign
   * http://cogcomp.cs.illinois.edu/
   */
-package edu.illinois.cs.cogcomp.saul;
+package edu.illinois.cs.cogcomp.saul.test;
 
```

Comment:
Side note for future: IMO this file's name is horrible. Should probably be `TestReal` (as apposed to `TestDiscrete`).